apps/fastlane/helpers/build_helper.rb

```diff
@@ -0,0 +1,132 @@
+require 'json'
+require 'xcodeproj'
+require_relative 'github_helper.rb'
+
+platform :android do
+  lane :build do |values|
+    build_notes = get_build_notes()
+    test_groups = get_build_test_groups()
+    app_package_name = CredentialsManager::AppfileConfig.try_fetch_value(:package_name) # get package_name from Appfile
+
+    UI.important(find_firebase_app_id(app_identifier: app_package_name))
+
+    # Build release APK using Flutter CLI
+    sh("flutter build apk --release")
```

Comment:
Do you suggest we build an APK compared to AAB?

Reply:
I did consider using AAB as well, but then switched to APK because I think it will be easier for anyone to download from Firebase and install the APK if needed, especially for non-registered devices (e.g., emulators).